### ISSUE TITLE
A recent compiled version (fixing UTF-8 text issues) for anyone who needs it

### ISSUE THREAD

**Issue by mnihyc** [xuan32546/IOS13-SimulateTouch#88]
https://github.com/mnihyc/ZXTouch/releases/tag/v0.0.8

**Comment by NguyenDuyThang** [comment]
Hello @mnihyc , the socket breaks then crashes when calling insert_text. This is useful, can you check it

**Comment by mnihyc** [comment]
> Hello @mnihyc , the socket breaks then crashes when calling insert_text. This is useful, can you check it

It works fine on my iPhone XR (iOS 14.4.1). I didn’t modify the source code itself, so if the original release works, this build should as well.

**Comment by NguyenDuyThang** [comment]
I tested on iOS 16.7.10 and it crashed, the document said that "minimal iOS version has increased to 14.0" so I thought it works on iOS 16.

**Comment by mnihyc** [comment]
> I tested on iOS 16.7.10 and it crashed, the document said that "minimal iOS version has increased to 14.0" so I thought it works on iOS 16.

Yeah that’s just because my Xcode version only supports building for iOS 14 and above. :lol
It’s currently working fine for me, but you might want to take some time to diagnose and upgrade it for higher iOS versions. Good luck!

**Comment by geekonion** [comment]
Thanks for your efforts! This version builds upon your work and has been modified to be compatible with Dopamine Jailbreak.
https://github.com/geekonion/ZXTouch

**Comment by epic0002** [comment]
ios 16 rootlesss support https://github.com/Epic0001/zxtouchrootless
